docs/tools/pizza-cli.md

```diff
@@ -1,76 +1,85 @@
 
-📦 Download & Install
+### 📦 Download and Install
+
 The pizza CLI provides multiple installation methods to cater to the varied preferences of its users:
 
-- Homebrew: For macOS users who prefer the Homebrew package manager:
+- **Homebrew**: For macOS users who prefer the Homebrew package manager.
+
+  ```bash
+  brew install open-sauced/tap/pizza
+  ```
+
+- **NPM**: For those who are fond of the Node Package Manager.
 
-```bash
-brew install open-sauced/tap/pizza
-```
+  ```bash
+  npm i -g pizza
+  ```
 
-NPM: For those who are fond of the Node Package Manager:
+- **Direct script install**: A quick way to get started by directly fetching the installation script from GitHub.
 
-```bash
-npm i -g pizza
-```
+  ```bash
+  curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh
+  ```
 
-Direct Script Install: A quick way to get started by directly fetching the installation script from GitHub:
+  ⚠️ **Warning**: Piping scripts directly from the web to your shell can be risky. Always ensure you trust the source. For a safer approach, download, inspect, and then execute the following commands:
 
-```bash
-curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh
-```
-⚠️ Warning: Piping scripts directly from the web to your shell can be risky. Always ensure you trust the source. For a safer approach, download, inspect, and then execute:
+  ```bash
+  curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh > install.sh
+  vim install.sh
+  ./install.sh
+  ```
 
-```bash
-curl -fsSL https://raw.githubusercontent.com/open-sauced/pizza-cli/main/install.sh > install.sh
-vim install.sh
-./install.sh
-```
-Manual Build and Install: For those who like a hands-on approach:
+- **Manual build and install**: For those who like a hands-on approach.
 
-```bash
-make install
-```
-Ensure /usr/local/bin/ is in your path with:
+  ```bash
+  make install
+  ```
 
-```bash
-export PATH="$PATH:/usr/local/bin"
-```
-Alternatively, after building with make build, move the binary to your desired location:
+  Ensure `/usr/local/bin/` is in your path with:
+
+  ```bash
+  export PATH="$PATH:/usr/local/bin"
+  ```
+
+  Alternatively, after building with `make build`, move the binary to your desired location:
+
+  ```bash
+  mv build/pizza <your-preferred-path>
+  ```
 
-```bash
-mv build/pizza <your-preferred-path>
-```
 ### 🖥️ Local Development Setup
+
 To contribute to or modify the pizza CLI, you need a specific set of tools:
 
-- Go Toolchain: The foundational tool for building and managing Go applications.
-- Docker: Required for linting and other auxiliary tooling.
-- Make: A build automation tool.
-Development Commands:
+- [**Go Toolchain**](https://go.dev/doc/install): The foundational tool for building and managing Go applications.
+- [**Docker**](https://docs.docker.com/engine/install/): Required for linting and other auxiliary tooling.
+- **Make**: A build automation tool.
```

Comment:
I added the links to "Go Toolchain" and "Docker" here based on our README in the [pizza-cli repo](https://github.com/open-sauced/pizza-cli).

It would be nice to also provide link for "Make" so that our users understand what we are using here. I'm not sure if what we use here is the [GNU Make](https://www.gnu.org/software/make/).

Later on, the link can be implemented as well  in the README of the pizza-cli repo.

Again, I need second eyes and opinion from our engineerings on this.

Reply:
This section was quickly added, so it definitely needs to be clarified. I'd like to add some gifs too bc it' a pretty cool tool.

Reply:
Awesome! We can later create a new issue to update this then. :)